mindsdb/interfaces/storage/db.py

```diff
@@ -462,6 +462,7 @@ class Agents(Base):
     project_id = Column(Integer, nullable=False)
 
     model_name = Column(String, nullable=True)
+    provider = Column(String, nullable=True)
```

Comment:
if we move `provider` from params we have to create migration for:
- adding this column to database
- move values from params to new column for existed agents

Reply:
Thanks! I added the migrations to git, forgot to add them before